docs/source/references.bib

```diff
@@ -3,7 +3,7 @@
 
 
 @Article{Altamimi2002,
-  Title                    = {{ITRF2000}: A new release of the {I}nternational {T}errestrial {R}eference {F}rame for earth science applications},
+  Title                    = {ITRF2000: A new release of the International Terrestrial Reference Frame for earth science applications},
```

Comment:
FYI, the curly bracket are needed by BibTeX to preserve the upper case letters. You can see the HTML/PDF output is now "Itrf2000: a new release of the international terrestrial reference frame" without the curly braces.

This isn't my favourite part of BibTeX either!

Reply:
I removed them because compiling the docs failed on me. It might have just been locally though, I can't really remember. Feel free to change it back!